### PR TITLE
chore: Adds assignee filter in sidebar for home screen

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
https://www.loom.com/share/adc1bd43c0cb46b89932805a1961c09e

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/components/ChatList.vue [hunk 1]
```diff
@@ -24,18 +24,27 @@
...
           />
           <chat-filter
             class="chat-status--filter"
+            :active-type="activeStatus"
             @statusFilterChange="updateStatusType"
           />
-          <sort-by-filter @changeSortByFilter="updateSortByFilterType" />
+          <sort-by-filter
+            :active-type="activeSortBy"
+            @changeSortByFilter="updateSortByFilterType"
```
Comment (nithindavid) [inline app/javascript/dashboard/components/ChatList.vue:40]: @iamsivin Can we change the prop name on filter components to `change` from `changeSortByFilter`? As they are already in the component scope, and there aren't multiple on change events.

Comment (nithindavid) [reply app/javascript/dashboard/components/ChatList.vue:40]: Please do this for othet components as well

File: app/javascript/dashboard/components/layout/sidebarComponents/SecondaryNavItem.vue [hunk 2]
```diff
@@ -361,4 +461,36 @@ export default {
...
+    color: var(--s-600);
+    height: var(--space-medium);
+    padding: var(--space-smaller);
+    margin: var(--space-small) 0 0 0;
+
+    &:hover {
+      background: var(--s-25);
+    }
+    &.is-active {
+      background: var(--s-50);
```
Comment (nithindavid) [inline app/javascript/dashboard/components/layout/sidebarComponents/SecondaryNavItem.vue:479]:
```suggestion
      background: var(--w-25);
```
Can we try primary color for active items and see if they are good?

**Review by nithindavid — CHANGES_REQUESTED** [review]

**Comment by sojan-official** [comment]
@nithindavid  @iamsivin  will we be taking this forward ?

**Comment by iamsivin** [comment]
> @nithindavid  @iamsivin  will we be taking this forward ?

@sojan-official We can close this for now once we found a better way of implementation in UI. 

**ref**: https://github.com/chatwoot/chatwoot/issues/6309

**Pull request closed without merging**